src/View/ViewVarsTrait.php

```diff
@@ -26,6 +26,22 @@
 {
 
     /**
+     * The name of default View class.
+     *
+     * @var string
+     */
+    public $viewClass = null;
+
+    /**
+     * View instance.
+     *
+     * Won't be set until after ViewVarsTrait::createView() is called.
+     *
+     * @var \Cake\View\View
+     */
+    public $_view;
```

Comment:
Creating view each time won't allowing doing things like `$this->getView()->layout = 'foo'` in action or beforeRender, as the instance would be overwritten when `$this->getView()` is called later in `Controller::render()`.